extension/pgtelemetry--1.5.sql--1.6.sql

```diff
@@ -0,0 +1,19 @@
+-- WAL telemetry
+
+create function wal_telemetry_create_or_select_record()
+returns pg_telemetry_wal_log language plpgsql as
+$$
+declare log_entry pg_telemetry_wal_log;
+begin
+    if pg_is_in_recovery() then
+       select * into log_entry from pg_telemetry_wal_log order by run_time desc limit 1;
+    else
+       delete from pgtelemetry.pg_telemetry_wal_log where timestamp < (current_timestamp - interval '1 month');
```

Comment:
Can you omit schema name `pgtelemetry.`. I doubt that the extension will be moved to another schema other than `pgtelemetry`. But it is better to have consistency (see `insert into` below).